uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OutlookApp.java

```diff
@@ -185,4 +185,28 @@ public boolean isSignInSnackBarPresent() {
         // Check if the sign in SnackBar is present
         return UiAutomatorUtils.obtainUiObjectWithResourceId("com.microsoft.office.outlook:id/snackbar_action", TimeUnit.SECONDS.toMillis(5)).exists();
     }
+
+    /**
+     * Check that outlook does not have an option for phone sign-up
+     * @return true if the option is not available, false otherwise
+     */
+    public boolean checkPhoneSignUpIsNotAvailable() {
```

Comment:
>checkPhoneSignUpIsNotAvailable [](http://example.com/codeflow?start=19&length=30)

Consider making this a method in IFirstPartyApp

Reply:
Reason i didn't is because not every first party app will use this, for example outlook and onedrive will be different